### PR TITLE
Edits after 1.3.0 alpha.1

### PR DESCRIPTION
**Pull request by maelvls**
I added this big warning:

warning

Also, I added a nice table to explain which variable is which:

table

@irbekrm @SgtCoDFish @wallrj

### CODE REVIEW

File: content/en/docs/contributing/release-process.md [hunk 1]
```diff
@@ -10,6 +10,31 @@ new release of cert-manager.
...
+   "Settings" tab when looking at the cert-manager project, you **are** an
+   owner). To become a GitHub "owner" of the cert-manager project, [open a
+   PR](https://github.com/jetstack/platform-board/pulls/new) with a link to
+   here.
+2. You need to be added as an "Owner" to the GCP project
+   [cert-manager-release](https://console.cloud.google.com/?project=cert-manager-release).
+   To check if you do have access, try opening [the Cloud Build
+   page](https://console.cloud.google.com/cloud-build?project=cert-manager-release).
+   To get "Editor" access to the GCP project, you can copy-paste [this PR
+   example](https://github.com/jetstack/platform-board/issues/373) into a new PR
```
Comment (munnerz) [inline content/en/docs/contributing/release-process.md:31]: I think this is not publicly accessible?

Comment (maelvls) [reply content/en/docs/contributing/release-process.md]: Oops, good point 😞

I added a warning about being at jetstack in order to ask for the permissions; does it make sense?

I would (really) prefer not to have this "must be at jetstack" thing. In my mind, we should be able to ask for this access as long as you are listed as a code owner of cert-manager. In this case, I think you should be able to request access by creating a PR on https://github.com/jetstack/cert-manager and have it reviewed by Zee (IT team at Jetstack).

What do you think?

File: content/en/docs/contributing/release-process.md [hunk 2]
```diff
@@ -10,6 +10,61 @@ new release of cert-manager.
...
+-->
+
+{{% pageinfo color="warning" %}}
+
+⛔️ Do not proceed with the release process if you do not satisfy with one of the
+following:
+
+1. The release process **takes about 40 minutes**. You must have time to
+   complete all the steps.
+2. You need to be at Jetstack to create ask for the GitHub and GCP permissions.
```
Comment (SgtCoDFish) [inline content/en/docs/contributing/release-process.md:24]: maybe change to:

> You currently need to be at Jetstack to get the required GitHub and GCP permissions.
> (we'd like contributors outside Jetstack to be able to get access; if that's of interest to you, please let us know)

File: content/en/docs/contributing/release-process.md [hunk 3]
```diff
@@ -10,6 +10,61 @@ new release of cert-manager.
 
 ## Prerequisites
 
+<!--
+About "alert" and "pageinfo": https://www.docsy.dev/docs/adding-content/shortcodes/#shortcode-helpers
+-->
+
+{{% pageinfo color="warning" %}}
+
+⛔️ Do not proceed with the release process if you do not satisfy with one of the
```
Comment (SgtCoDFish) [inline content/en/docs/contributing/release-process.md:19]: > Do not proceed with the release process if you do not meet all of the following conditions:

File: content/en/docs/contributing/release-process.md [hunk 4]
```diff
@@ -10,6 +10,61 @@ new release of cert-manager.
...
+   PR](https://github.com/jetstack/platform-board/pulls/new) with a link to
+   here.
+4. You need to be added as an "Owner" to the GCP project
+   [cert-manager-release](https://console.cloud.google.com/?project=cert-manager-release).
+   To check if you do have access, try opening [the Cloud Build
+   page](https://console.cloud.google.com/cloud-build?project=cert-manager-release).
+   To get the "Editor" permission on the GCP project, copy-paste the below
+   example template PR into a [new
+   PR](https://github.com/jetstack/platform-board/pulls/new) if you want to get
+   the permissions.
```
Comment (SgtCoDFish) [inline content/en/docs/contributing/release-process.md:39]: remove: "if you want to get the permissions"

File: content/en/docs/contributing/release-process.md [hunk 5]
```diff
@@ -10,6 +10,61 @@ new release of cert-manager.
...
+
+   <summary>
+   📝 PR template for requesting access to the GCP project
+   </summary>
+
+   ```markdown
+   Step 4 under "Prerequisites" on the [release-process][1] page means I need
+   access to the [cert-manager-release][2] project on GCP.
+
+   I would need to be an "Editor" on that project. More specifically, the roles
```
Comment (SgtCoDFish) [inline content/en/docs/contributing/release-process.md:51]: > I need to be an Editor on this project. More specifically, the roles I need are:

File: content/en/docs/contributing/release-process.md [hunk 6]
```diff
@@ -111,30 +154,66 @@ The process for cutting a minor release is as follows:
...
 
      ```bash
+     # Must be run from the cert-manger repo folder.
      git push --set-upstream origin release-1.0
      ```
 
 4. Generate and edit the release notes:
 
-   1. Run as follows, substituting the current and last versions where appropriate:
+   1. Use the following two tables to understand how to fill in the 4
```
Comment (SgtCoDFish) [inline content/en/docs/contributing/release-process.md:163]: Might be helpful to add a link to the k8s docs for the tool? https://github.com/kubernetes/release/blob/f69aaae60a036a7d33c120b3f0e8ad565646aa75/cmd/release-notes/README.md#options

File: content/en/docs/contributing/release-process.md [hunk 7]
```diff
@@ -111,30 +154,66 @@ The process for cutting a minor release is as follows:
...
+
+        ```sh
+        export START_REV="v1.2.0"
+        export END_REV="release-1.3"
+        export BRANCH="release-1.3"
+        export RELEASE_VERSION="1.3.0-alpha.0"
+        ```
 
-        ```bash
+   2. Generate the `release-notes.md` at the root of your the cert-manager repo
```
Comment (SgtCoDFish) [inline content/en/docs/contributing/release-process.md:200]: > Generate `release-notes.md` at the root of your cert-manager repo

(remove both "the"s)

File: content/en/docs/contributing/release-process.md [hunk 8]
```diff
@@ -183,19 +261,33 @@ The process for cutting a minor release is as follows:
...
 6. Run `cmrel publish`
 
-   1. First do a dry-run, to ensure that all the staged resources are valid.
+   1. Find the "release name". The release name can be found in the output of
+      the previous command, `cmrel stage`. In the previous command output, look
+      for a line that looks like this:
 
-        ```bash
-        cmrel publish --release-name <RELEASE_NAME>
+        ```sh
```
Comment (SgtCoDFish) [inline content/en/docs/contributing/release-process.md:268]: this is `text` now, not `sh`

File: content/en/docs/contributing/release-process.md [hunk 9]
```diff
@@ -183,19 +261,33 @@ The process for cutting a minor release is as follows:
...
         ```
 
-        Where `<RELEASE_NAME>` is the unique build ID printed by the earlier `cmrel stage` command.
+        Copy the part that starts with a version (`v1.3.0`). That's the "release
+        name" that you will need for the next step:
 
-        For example, given the bucket
-        `gs://cert-manager-release/stage/gcb/release/v1.0.0-219b793`, the
-        `RELEASE_NAME` would be `v1.0.0-219b793`.
+        ```sh
```
Comment (SgtCoDFish) [inline content/en/docs/contributing/release-process.md:275]: again, not `sh`

**Review by SgtCoDFish — CHANGES_REQUESTED** [review]
A few comments on some of the changes; overall looks like a great improvement

File: content/en/docs/contributing/release-process.md (hunk 2 above)
Comment (maelvls) [reply content/en/docs/contributing/release-process.md]: Thanks! Fixed in 2909939 👍

File: content/en/docs/contributing/release-process.md (hunk 3 above)
Comment (maelvls) [reply content/en/docs/contributing/release-process.md]: Thanks! Fixed in 2909939 👍

File: content/en/docs/contributing/release-process.md (hunk 4 above)
Comment (maelvls) [reply content/en/docs/contributing/release-process.md]: Thanks! Fixed in 2909939 👍

File: content/en/docs/contributing/release-process.md (hunk 5 above)
Comment (maelvls) [reply content/en/docs/contributing/release-process.md]: Thanks! Fixed in 2909939 👍

File: content/en/docs/contributing/release-process.md (hunk 6 above)
Comment (maelvls) [reply content/en/docs/contributing/release-process.md]: Thanks! Fixed in 2909939 👍

File: content/en/docs/contributing/release-process.md (hunk 7 above)
Comment (maelvls) [reply content/en/docs/contributing/release-process.md]: Thanks! Fixed in 2909939 👍

File: content/en/docs/contributing/release-process.md (hunk 6 above)
Comment (maelvls) [reply content/en/docs/contributing/release-process.md]: Done in 8f9273fd6c45b3819f7563cf7245da5e02a504ae! thanks!

File: content/en/docs/contributing/release-process.md (hunk 7 above)
Comment (maelvls) [reply content/en/docs/contributing/release-process.md]: Done in 8f9273fd6c45b3819f7563cf7245da5e02a504ae! thanks!

File: content/en/docs/contributing/release-process.md (hunk 8 above)
Comment (maelvls) [reply content/en/docs/contributing/release-process.md]: Done in 8f9273fd6c45b3819f7563cf7245da5e02a504ae! thanks!

File: content/en/docs/contributing/release-process.md (hunk 2 above)
Comment (maelvls) [reply content/en/docs/contributing/release-process.md]: Thanks! Fixed in da4b1e94c8380d3d9d9cdabf75f88aaa820564ba

File: content/en/docs/contributing/release-process.md (hunk 3 above)
Comment (maelvls) [reply content/en/docs/contributing/release-process.md]: Thanks! Fixed in da4b1e94c8380d3d9d9cdabf75f88aaa820564ba

File: content/en/docs/contributing/release-process.md (hunk 5 above)
Comment (maelvls) [reply content/en/docs/contributing/release-process.md]: Thanks! Fixed in da4b1e94c8380d3d9d9cdabf75f88aaa820564ba

File: content/en/docs/contributing/release-process.md (hunk 6 above)
Comment (maelvls) [reply content/en/docs/contributing/release-process.md]: Thanks! Fixed in da4b1e94c8380d3d9d9cdabf75f88aaa820564ba

File: content/en/docs/contributing/release-process.md (hunk 7 above)
Comment (maelvls) [reply content/en/docs/contributing/release-process.md]: Thanks! Fixed in da4b1e94c8380d3d9d9cdabf75f88aaa820564ba

File: content/en/docs/contributing/release-process.md [hunk 10]
```diff
@@ -48,7 +47,7 @@ following:
    Step 4 under "Prerequisites" on the [release-process][1] page means I need
    access to the [cert-manager-release][2] project on GCP.
 
-   I would need to be an "Editor" on that project. More specifically, the roles
+   I need to be an "Editor" on this project. More specifically, the roles I need are:
    that I will be using are:
```
Comment (SgtCoDFish) [inline content/en/docs/contributing/release-process.md:51]: This line needs removing

File: content/en/docs/contributing/release-process.md [hunk 11]
```diff
@@ -160,8 +159,10 @@ The process for cutting a minor release is as follows:
 
 4. Generate and edit the release notes:
 
-   1. Use the following two tables to understand how to fill in the 4
-      environment variables needed for the next step:
+   1. Use the following two tables to understand how to fill in the four
+      environment variables needed for the next step. These four environment
+      variables are documented on the [release-note's
+      README](https://github.com/kubernetes/release/blob/master/cmd/release-notes/README.md#options).
```
Comment (SgtCoDFish) [inline content/en/docs/contributing/release-process.md:165]: I think this should be:

> "documented on the [README](https://github.com/kubernetes/release/blob/master/cmd/release-notes/README.md#options) for the Kubernetes `release-notes` tool.

**Review by SgtCoDFish — CHANGES_REQUESTED** [review]
A couple more minor things

File: content/en/docs/contributing/release-process.md (hunk 10 above)
Comment (SgtCoDFish) [reply content/en/docs/contributing/release-process.md:51]: Specifically, "that I will be using are: "

Comment (maelvls) [reply content/en/docs/contributing/release-process.md:51]: Thanks! Fixed in 5e1bc1d

File: content/en/docs/contributing/release-process.md [hunk 12]
```diff
@@ -258,33 +260,35 @@ The process for cutting a minor release is as follows:
...
 6. Run `cmrel publish`
+   
+   <div class="p-3 display-4">Background: Primary</div>
 
-   1. Find the "release name". The release name can be found in the output of
-      the previous command, `cmrel stage`. In the previous command output, look
-      for a line that looks like this:
+   1. Set the `RELEASE_NAME` variable in your shell by. The value for the
+      `RELEASE_NAME` variable is found in the output of the previous command,
+      `cmrel stage`. Look for the line that contains the `gs://` link:
```
Comment (SgtCoDFish) [inline content/en/docs/contributing/release-process.md:273]: "by" in the first line needs removing:

> Set the `RELEASE_NAME` variable in your shell.

**Review by SgtCoDFish — CHANGES_REQUESTED** [review]
Just one minor thing

File: content/en/docs/contributing/release-process.md (hunk 12 above)
Comment (maelvls) [reply content/en/docs/contributing/release-process.md:284]: Thanks again 😅 Fixed in 26443c8
I should proof-read everything I write at least once 😇

File: content/en/docs/contributing/release-process.md (hunk 1 above)
Comment (maelvls) [reply content/en/docs/contributing/release-process.md]: If that's OK for you, I'll proceed with merging this PR

Feel free to let me know here if you feel we should rephrase this or dig deeper into how to make the process friendly to people external to jetstack (I'll raise this issue during tonight's biweekly dev meeting)

**Review by SgtCoDFish — APPROVED** [review]
/lgtm

**Pull request merged**